### PR TITLE
Add using list

### PR DESCRIPTION
**Pull request by chetan639**

### CODE REVIEW

File: using-list.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+myStr = ['H','e','l','l','o',', ','W','o','r','l','d','!']
+
+for s in myStr:
+    print(s,end='')
+print()
```
Comment (Goku-kun) [inline using-list.py:5]: Remove the redundant print() and instead have the last line as an empty new line.
```suggestion

```

Comment (Goku-kun) [reply using-list.py:5]: Wow. I can't believe list iteration was yet to be implemented. Good find.

I'll be accepting this pull request as soon as you can make the requested changes.

**Review by Goku-kun — CHANGES_REQUESTED** [review]

File: using-list.py (hunk 1 above)
Comment (chetan639) [reply using-list.py]: Doing it right away!

**Comment by chetan639** [comment]
I have removed the unnecessary print!

**Review by Goku-kun — APPROVED** [review]
Cool. Looks good  to me.

**Pull request merged**